assets/arti.qrc

```diff
@@ -11,11 +11,6 @@
     <file>imgs/arti/main/main.png</file>
     <file>imgs/arti/main/pmod.png</file>
     <file>imgs/arti/main/sdoe.png</file>
-    <file>imgs/arti/acpt/ccol.webp</file>
-    <file>imgs/arti/acpt/fwol.webp</file>
-    <file>imgs/arti/acpt/gboe.webp</file>
-    <file>imgs/arti/acpt/pmod.webp</file>
-    <file>imgs/arti/acpt/sdoe.webp</file>
```

Comment:
Please note that these are not repeated entries but rather the ones from the artifact set "Archaic Petra". As such, these do not require any removal unless these have been referenced somewhere else already that you can point me to.